contents/handbook/strategy/roadmap.md

```diff
@@ -4,66 +4,48 @@ sidebar: Handbook
 showTitle: true
 ---
 
-## Up-to-date roadmap
+Our mission is to make more successful products in the world. 
 
-We keep our most up-to-date roadmap on [GitHub](https://github.com/orgs/PostHog/projects/1).
+Our roadmap for 2021 will do three things:
+1. Create a solid core product that's easy to use
+2. Have the best developer platform for event-based analytics
+3. Set us up to service huge volumes
 
-## General Product Strategy
+# 1. Core product
 
-### Open Core Business Model
+PostHog is a product that people love, primarily because covers 90% of analytics use cases but bundled into one package.
+Some examples of the functionality we've built last year:
 
-We aim for every company that builds software to be using PostHog.
+- Product analytics
+- Session recording
+- Feature flags
+- Heatmaps
+- Autocapture
 
-* The open source platform is designed to be installed by a developer when they start a new project, before a Product Team is in place. There is no competition at this stage.
-* The open source platform is also well suited to adoption in an existing startup, midmarket or enterprise customer as it has no information security and no vendor risk management requirements. We are much lower friction.
-* Providing self-hosted capabilities means we have complete flexibility over how we charge customers. This means we can align with higher volume customers.
+There's plenty of work to be done within those categories to make a product that is especially useful for engineers and other product minded people. To add on to that, this year we want to build:
 
-### Becoming a Piece of Infrastructure
+- A/B testing
+- User feedback
```

Comment:
I'm a strong fan of this. I know it sounds dull technically, but for the purpose of getting to product market fit, I think just being able to replace a crappy notion database, and having usage data next to the notes would be really powerful.